GitHub Open Source Community Standards

svengreb opened the pull request:
> Closes #9 
Blocks #19 (must be merged **before**)

svengreb committed:
Adapt to the GitHub Open Source community standards

Adapted to the recommended GitHub community standards (1) using the
Open Source Guide (2) to complete the projects community profile (3).

Read the introduction blog posts "Code of Conduct" (4) and "Contribution
Guidelines" (5) for more details.

References:

  (1) https://github.com/blog/2380-new-community-tools
  (2) https://opensource.guide
  (3) https://github.com/communicode/communikey-backend/community
  (4) https://help.github.com/articles/adding-a-code-of-conduct-to-your-project
  (5) https://github.com/blog/1184-contributing-guidelines

GH-9

fermentfan commented on CONTRIBUTING.md:
```diff
@@ -1,92 +1,205 @@
+The project development workflow and process uses [GitHub Issues][gh-issues]- and [Pull Requests][gh-pr] management to track issues and pull requests.
 
-### Our Standards
+Before you continue with this contribution guideslines we highly recommend to read the awesome GitHub [Open Source Guide](https://opensource.guide) on how to [making open source contributions][gh-osguide-contribute].
```
> Before you continue with **these** contribution **guidelines**

fermentfan commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,56 @@
+
+## What we believe in and how we act
+
+* In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation. ...
```
>  we as contributors and maintainers pledge to **make** participation

fermentfan commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,56 @@
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project core team members or owner via [email][email]. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Furth ...
```
Add our communicode email addresses?

svengreb replied:
The main idea of this (broken) link was to add a support email address like e.g. communikey-support@communicode.de, but I think we should start by adding a [`.mailmap`][git-mm] file and link to this instead. This prevents unnecessary overhead of updating all documents when new team members are added and additionally adds the main functionality of the file: Mapping commits when someone uses a different email address.

[git-mm]: https://git-scm.com/docs/git-shortlog#_mapping_authors

fermentfan reviewed (changes requested):
Fix grammar

svengreb committed:
Fix typos and grammar from code review requests

GH-9

svengreb committed:
Add Git mail mapping file

Added a .mailmap (1) file to link to in documentations to allow
contributors to send mails regarding security issues. This prevents
unnecessary overhead of updating all documents when new team members
are added and additionally adds the main functionality of the file:
Mapping commits when someone uses a different email address.

References:

 (1) https://git-scm.com/docs/git-shortlog#_mapping_authors

GH-9
